library/src/test/java/kwasm/format/text/LimitsTest.kt

```diff
@@ -122,13 +122,6 @@ class LimitsTest {
         assertThat(exception).hasMessageThat().contains("Illegal value")
     }
 
-    @Test
-    fun parseTwoValuesWithNegativeMax_throwsParseExceptionWithNegativeNumberMessage() {
-        val tokens = tokenizer.tokenize("123456 -234567", context)
-        val exception = assertThrows(ParseException::class.java) { tokens.parseLimits(0) }
-        assertThat(exception).hasMessageThat().contains("Expected integer literal")
```

Comment:
I think this is a valid test case to keep around, but assert that it only gives you a limit with the first value.